src/OrchardCore.Modules/OrchardCore.ContentFields/Drivers/MultiTextFieldDisplayDriver.cs

```diff
@@ -10,7 +10,7 @@
 using OrchardCore.DisplayManagement.ModelBinding;
 using OrchardCore.DisplayManagement.Views;
 using OrchardCore.Mvc.ModelBinding;
-using OrchardCore.Mvc.Utilities;
+using OrchardCore.ContentManagement.Utilities;
```

Comment:
Then not at the right place ;)

Ordered alphabetically unless System namespaces.

Under Visual Studio you can right click and do `Remove and Sort Usings`, or Ctrl + R + G

For info, for some basic formatting Ctrl + K + D